### PR TITLE
✨ Add Match components

### LINKED ISSUE #22: Match container
Show game, username and number of highlights

### LINKED ISSUE #25: Implement match details
Wireframe see [Match Highlights](https://www.figma.com/file/IrLlOYxHznbvyD2uYIQNny/Version-1?node-id=4%3A47)

### PR DESCRIPTION
**Pull request by lmachens**
Very basic implementation based on the wireframes.
fix #22 
fix #25 

image
image

If we have a design, this component will be updated. 
In addition, we could think about adding a link to this component.

The game title is loaded from the Overwolf API based on the `gameClassId`.

**Breaking**
I change global style for `p` elements.

### CODE REVIEW

**Review by AlexisRoe — APPROVED** [review]
gj

File: src/app/components/MatchItem/MatchItem.module.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.container {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: space-between;
```
Comment (AlexisRoe) [inline src/app/components/MatchItem/MatchItem.module.css]: You use this three properties two times in this pr, would it be efficient to put this into a global css file and attach both classes to the html elements?

**Review by AlexisRoe — APPROVED** [review]
thanks, looks good to me

**Review by webdevelukas — APPROVED** [review]
lgtm 🏆

**Pull request merged**